pkgs/development/python-modules/yoto-api/default.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytz,
+  requests,
+  paho-mqtt,
+}:
+
+buildPythonPackage rec {
+  pname = "yoto-api";
+  version = "1.24.2";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "cdnninja";
+    repo = "yoto_api";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-wbqeRTYHDpxKqOwTRkkYGtgYOmnyNQY50dnvI0WprwA=";
+  };
+
+  propagatedBuildInputs = [
+    pytz
+    requests
+    paho-mqtt
+  ];
+
```

Comment:
```
  nativeCheckInputs = [ unittestCheckHook ];
```

Run the test suite. Might also be compatible with `pytestCheckHook`.

Reply:
The unittests require connection to the Yoto API. I don't think it will be possible to run the test suite:
- https://github.com/cdnninja/yoto_api/blob/master/README.rst#tests

Reply:
All of them? With `pytestCheckHook` you can use `disabledTests` and `disabledTestsPaths` to reduce the tests we run.